benches/suites/raw_node.rs

```diff
@@ -20,3 +25,107 @@ pub fn bench_raw_node_new(c: &mut Criterion) {
 
     c.bench_function("RawNode::new", bench);
 }
+
+pub fn bench_raw_node_leader_propose(c: &mut Criterion) {
+    static KB: usize = 1024;
+    let mut test_sets = vec![
+        0,
+        32,
+        128,
+        512,
+        KB,
+        4 * KB,
+        16 * KB,
+        128 * KB,
+        512 * KB,
+        KB * KB,
+    ];
+    let mut group = c.benchmark_group("RawNode::leader_propose");
+    for size in test_sets.drain(..) {
+        // Calculate measurement time in seconds according to the input size.
+        // The approximate time might not be the best but should work fine.
+        let mtime = if size < KB {
```

Comment:
Maybe it's better to just use `7` for everything?

Reply:
I'm worried that the total measurement time will be too long. We can still get enough analysing samples from the benchmark with tiny throughput by using shorter benching time.